docs/notebooks/algorithms/spline_fxns_tutorial.ipynb

```diff
@@ -0,0 +1,312 @@
+{
```

Comment:
My only feedback is I think it would be useful if you added more comments/text explaining the experiment and the results. Higher degree seems to lead to more accurate measurements, why? Is the highest degree estimate exactly correct (theoretically)?

Reply:
Addressed in new PR